MAINTAINERS.md

```diff
@@ -0,0 +1,101 @@
+the project. A Kubernetes maintainer is a contributor who has contributed/is
+contributing to the project regularly and a developer that demonstrated good
+technical judgement. The following is a list of requirements to be a Kubernetes
+maintainer.
+
+Before continuing, it should be noted that we value every contributions, however
+small it may be, even a comment on an issue a valuable contribution to us.
+Maintainership requires time commitment and community trust. You don't have to
+be a maintainer to have a meaningful impact on Kubernetes!
+
+## How to become a maintainer
+
+### 1. Be an active contributor
+
+An active contributor is someone who contributed regularly in the last 2 months
+and is one of the top 20 of contributors for the last 2 months.
+
+For example in the context of merged commit contributions, from September to
+November, the 20th developer has 20 commits.
+
+#### Contribution Types
+
+Contributions are not limited to code. Other contributions include PR reviews,
+discussion participations, test ownership, SIG leadership, etc. At the moment we
+are only able to measure PR contributions. As we improve our project tooling,
+we'll be able to measure and analyze other types of contributions and take those
+into account when evaluating a maintainer application.
+
+### 2. Be experienced in the project
+
+You need to have more than 50 contributions total and project participation for
+more than a year in the Kubernetes organization.
+
+### 3. Find 3 senior maintainer sponsors
+
+As you contribute to Kubernetes, you'll be collaborating with different
+community members. Having the approval of senior project maintainers means that
+they have seen your work and they believe that you'll be a good addition to the
+project. 
+
+**Motivation:** We want the sponsorships to be meaningful, so we require the sponsors to be
+maintainers that collaborated with you and we also don't want to burden the
+senior maintainers otherwise they'll be reluctant to sponsor anyone. Therefore,
+rather than having a mentorship process where a new maintainer is guided by a
+senior member, we want to see the capabilities of the candidate before being
+accepted.
+
+**TODO:** Detail the meaning of a senior project maintainer with a requirement
+along the lines of more than 6 months experience as a project maintainer.
+
+### 4. Apply for maintainership to contributor leads
+
+The application will be discussed by the lead maintainers and put to vote. If
+the vote result is negative, do not be discouraged! You will be given feedback
+so that you can improve yourself and apply again. We want to empower our
+community and cultivate a trust relationship that's why we carefully evaluate
+abilities and experience of each candidate.
+
+**TBD:** As of now Brian Grant is the wg-contribex lead that approves new maintainers.
+
+## What we expect from our maintainers
+
+First and foremost, be a good person, as cheeky as it sounds, you are now a
```

Comment:
@resouer 

Approvers will be able to approve via bot command.

We have many roles we will need to document.

Copied from the group thread:

* New contributor: a couple of PRs
* Contributor: more than a couple PRs (which could be docs as well as code)
* Org member: active enough to be useful to assign issues to them and add them to a github team (e.g., for a SIG) for notification purposes; if they choose public membership, they get a badge on their github profile
* kubernetes-collaborators: "read access" to kubernetes repo; get a badge on PR and issue comments; trusted enough to run tests on their PRs automatically; can issue "@k8s-bot ok to test" for other contributors
* Reviewer: In some OWNERS file as a reviewer (in repos using the bot), assigned related PRs, assigned relevant test bugs; can champion incubator repos
* Approver (new): In some OWNERS file as an approver; will be needed to get code merged
* SIG participant: active in one or more areas of the project; wide variety of roles are represented
* SIG lead: SIG organizer
* Area/component owner: design/proposal approval authority for some area of the project, though escalation is still possible, and most beta/GA API changes are vetted by the API owners
* API owners: lead designers of the project, who are familiar with the design, requirements, mechanics, conventions, style, scope, gotchas, etc. of the API
* Team lead: tech lead or manager of some team at some company working on K8s; can influence priorities of their team members; pragmatically, probably want label/assignment powers
* Top-level kubernetes/OWNERS: de-facto project elders; technically can approve virtually any PRs; can sponsor incubator repos
* kubernetes-maintainers: write access to repo (assign issues/PRs, add/remove labels and milestones, edit issues and PRs, edit wiki, create/delete labels and milestones); technically can lgtm any PR and cause it to be merged by the submit queue; expected to review PRs and fix bugs related to their domains
* kubernetes-pm github team: manage issues
* kubernetes-pm googlegroup: product/project management, release/feature planning and curation
* kubernetes-admin: direct code write/merge access; for build cops and release czars only
* Build cops: ensure tests pass, submit queue is working, rollback PRs, manually merge as necessary to fix build
* User-support rotation: answer questions on stackoverflow, googlegroups, slack, twitter, etc. full time while on duty
* Minor release Czar: drive minor release
* Patch release Czar: manage cherrypick and patch release process
* K8s Org owner: can create repos, do ~any github action; should remain O(10) people very familiar with project workings and distributed across a few time zones and organizations 

The other repos will have distinct sets of people filling some of the above roles, also.